docs/Developer Reference/contributing-to-chainlink.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: nodes.liquid
+section: smartContract
+date: Last Modified
+title: "Contributing to Chainlink"
+permalink: "docs/contributing-to-chainlink/"
+---
+Chainlink is an open-source project licensed [under the MIT license](https://github.com/smartcontractkit/chainlink/blob/master/LICENSE), and we encourage contributions from all developers and community members.
+
+[insert YT video here]
+
+# What It Means to Contribute
+To contribute to the Chainlink project means you as a developer or community member contribute your time and effort to help improve and grow Chainlink via various methods such as:
+- Build and maintain the Chainlink software and tools
+- Improving and maintaining the documentation, including translations into other languages
+- Creating Chainlink focused content (blog posts, tutorials, videos etc)
+- Becoming a developer expert
+- Running an in-person meetup or watch party
+- Running a Chainlink focused developer Bootcamp (in person or online)
+- Participate in a hackathon
+- Applying for a grant
+
+# Why Should You Contribute
+
+[Open source software](https://en.wikipedia.org/wiki/Open-source_software) is a model that brings multiple benefits for both the project as well as the contributors. As a developer or community member, contributing to Chainlink helps you to gain valuable skills and experience, improve the software you and many others rely on and use, as well as grow your personal brand in the community which can lead to future employment opportunities. On top of these awesome things, contributing to open source is fun,  gives you a real sense of community involvement, and gives you a real personal sense of satisfaction knowing that you're part of something bigger, that's helping to build something that will enable a fairer, more transparent and efficient new world.
```

Comment:
```suggestion
[Open source software](https://en.wikipedia.org/wiki/Open-source_software) is a model that brings multiple benefits for both the project and the contributors. As a developer or community member, contributing to Chainlink helps you to gain valuable skills and experience, improve the software that you use, and grow your personal brand in the community which can lead to future employment opportunities. On top of these awesome things, contributing to open source is fun.  It can give you a sense of community involvement, and gives you a personal sense of satisfaction knowing that you're part of an effort to build something that will enable a fairer, more transparent, and efficient new world.
```